### ISSUE TITLE
How can i add margin between preview images in Carousel.

### ISSUE THREAD

**Issue by danurasenan** [Ks89/angular-modal-gallery#226]
i want to add margin to the preview images in Carousel model like i show in the attached image. 

can someone help me with that. 

![Capture](https://user-images.githubusercontent.com/78210064/121228808-d6e10a00-c8aa-11eb-8cca-77c641ec7f8e.JPG)

**Comment by Ks89** [comment]
Hi. At the moment this is not possible because carousel has a fixed layout. Custom carousel layouts are not implemented. I'll add this issue as a feature for next releases. Thank you for your interest.

**Comment by danurasenan** [comment]
Hi @Ks89,

thank you for your response,

Is there a way that i can add boarder around image. because when i try to add black and white photos,  In previews its not showing separately.

Please check the attached Image.


![Capture](https://user-images.githubusercontent.com/78210064/121393818-58e63700-c96e-11eb-9e9f-81943b810836.JPG)

**Comment by Ks89** [comment]
Yes, you are right, with images of the same color it looks pretty bad.
However, at the moment this is not supported.

**Comment by danurasenan** [comment]
Hello, did you add the feature to add the margin between Carousel images

**Comment by Ks89** [comment]
Hi, no, sorry.
It's scheduled for v10.0.0.
At the moment, I'm very busy and I cannot schedule a release date for version 10.0.0. As soon I found some weeks of spare time to work on this I'll try to implement it. I cannot promise anything right now, sorry 😢 

However, if someone want to implement this with a pull request, feel free to contribute. There are two possibile solutions:
- implement this feature in a backward compatible mode (i don't know how) to release it as 9.1.0 (quicker to release, but probably more difficult to implement)
- implement this feature adding a breaking change, so it must be released as 10.0.0 (I still don't know when, so it will require many months)

**Comment by FrontEndProfile** [comment]
> i want to add margin to the preview images in Carousel model like i show in the attached image.
> 
> can someone help me with that.
> 
> ![Capture](https://user-images.githubusercontent.com/78210064/121228808-d6e10a00-c8aa-11eb-8cca-77c641ec7f8e.JPG)

yes , i think add animation's little bit :)
